### DISCUSSION TITLE
Updated to latest OpenAI library

### DISCUSSION THREAD

**Discussion by machinewrapped** [machinewrapped/llm-subtrans#82] [Announcements]
Updated the code to be compatible with the latest version of OpenAI's Python integration library, plus various bug fixes and UI improvements.

Hopefully I will be able to get builds that work for both PC and MacOS for this release!

<hr /><em>This discussion was created from the release <a href='https://github.com/machinewrapped/gpt-subtrans/releases/tag/v0.4.6'>Updated to latest OpenAI library</a>.</em>

**Comment by n-endashi** [comment]
I'm getting an error: Sunday, 08:57:07 - Unable to retrieve available AI models: module 'openai' has no attribute 'OpenAI' It doesn't seem to work at all now.

**Comment by machinewrapped** [comment]
@n-endashi you need to update the OpenAI library, they made breaking changes in the latest version. 

See this issue:
https://github.com/machinewrapped/gpt-subtrans/issues/83
